views/layouts/main.hbs

```diff
@@ -24,18 +24,19 @@
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.8.0/css/tachyons.min.css"/>
     <link rel="stylesheet" href="/css/main.css">
 </head>
-<body class="relative">
```

Comment:
This was added for the fork ribbon, does it still sit properly with this removed?

Reply:
Yes, it is not needed, it is correctly displayed without this class.